sktree/tree/tests/test_tree.py

```diff
@@ -4,9 +4,25 @@
 from sklearn.datasets import make_blobs
 from sklearn.metrics import adjusted_rand_score
 from sklearn.utils.estimator_checks import parametrize_with_checks
+from itertools import product, chain
+from sklearn import datasets
+from sklearn.metrics import accuracy_score
 
 from sktree.tree import UnsupervisedDecisionTree
 
+CLF_CRITERIONS = ("twomeans", "fastbic")
+
+CLF_TREES = {
+    "UnsupervisedDecisionTree": UnsupervisedDecisionTree,
+}
+
+# also load the iris dataset
+# and randomly permute it
+iris = datasets.load_iris()
+rng = np.random.RandomState(1)
+perm = rng.permutation(iris.target.size)
+iris.data = iris.data[perm]
+iris.target = iris.target[perm]
 
 @parametrize_with_checks([UnsupervisedDecisionTree(random_state=12)])
 def test_sklearn_compatible_estimator(estimator, check):
```

Comment:
Please add a parametrization to check `UnsupervisedDecisionTree(..., criterion='fastbic')` and explicitly make the existing one `UnsupervisedDecisionTree(..., criterion='twomeans')` here too. The default currently is "twomeans", but we might change that to "fastbic".

Reply:
See: https://scikit-learn.org/stable/modules/generated/sklearn.utils.estimator_checks.check_estimator.html

Reply:
Added!